Cython/Compiler/TypeSlots.py

```diff
@@ -932,38 +914,28 @@ def __init__(self, old_binops):
             BinopSlot(bf, "nb_add", "__add__", method_name_to_slot),
             BinopSlot(bf, "nb_subtract", "__sub__", method_name_to_slot),
             BinopSlot(bf, "nb_multiply", "__mul__", method_name_to_slot),
-            BinopSlot(bf, "nb_divide", "__div__", method_name_to_slot,
```

Comment:
Now that we have a warning, why not warn about `__div__`, `__oct__` etc. as well? If users choose Cython 3.1, it's because they target Python versions where these are useless. A warning would point at legacy code that users can remove on their side.

Reply:
It's a little harder because it's the alternative that generates the warning and these don't have alternatives. I decided it probably wasn't worth the effort for this.